packages/scripts/src/new/lib.ts

```diff
@@ -173,7 +123,7 @@ export async function main(): Promise<void> {
   copyFiles(inputs.type, inputs.n)
 
   log(blue.bold('Regenerating tsconfig and lego files'))
```

Comment:
Think we need to update the log here
```suggestion
  log(blue.bold('Regenerating tsconfig'))
```